tsconfig.json

```diff
@@ -0,0 +1,58 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
+    "target": "ES2017",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', or 'ESNEXT'. */
+    "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+    // "lib": [],                             /* Specify library files to be included in the compilation:  */
+    "allowJs": true,                       /* Allow javascript files to be compiled. */
```

Comment:
if you're going to have block comments on each line, line them all up with each other. Applies to this whole file.